docs/glossary.md

```diff
@@ -312,7 +312,7 @@ Corepack
     In practical terms, Corepack lets you use {term}`Yarn`, {term}`npm`, and {term}`pnpm` without having to install them.
 
     Corepack is distributed by default with all recent Node.js versions.
-    Run `corepack enable` to install the required Yarn and pnpm binaries on your path.
+    Run `npm i -g corepack@latest && corepack enable` to install the required Yarn and pnpm binaries on your path.
```

Comment:
We should never have put this command here because we don't know where the user would be in any given context.

```suggestion
```